### DISCUSSION TITLE
How to disposition issues

### DISCUSSION THREAD

**Discussion by huornlmj** [aquasecurity/trivy#4198] [Q&A] [answered]
I run `$ trivy config .` to scan my current and sub directories for misconfiguration analysis.  I get a report of multiple issues across multiple files.  I find that one particular rule (e.g. ds002 / "HIGH: Specify at least 1 USER command in Dockerfile with non-root user as argument") is found in two different files, file A and B.  I know that in file A it is a true positive issue but in file B it is not.  I want to dispose the ds002 finding in file B for this and future scans.  How do I actually do this?

**Comment by simar7** [comment]
sorry what do you mean by dispose?

**Comment by huornlmj** [comment]
Issue disposition.  I think I explained it pretty clearly in the initial comment.  Some might call it issue triage.  In short, an issue that will not be fixed, but Trivy is in a Ci/Cd pipeline and unless something is done it will always throw an error for the finding.  Most CLI security tools or linters provide a way to suppress issues, either inline or with config.  But the trivy config doesn't appear to provide granular control as needed here.

**Comment by itaysk** [comment]
@simar7 I believe in our terminology it's ignoring/acknowledging findings.
@homoluctus there are several ways and I agree they weren't clearly documented. We recently did a reorg on the documentation, and perhaps now it's a bit clearer. In any case, @AnaisUrlichs is working on clearer documentation on this subject exactly so I'm tagging her here as well.

**Comment by AnaisUrlichs** [comment, marked as answer]
Yes, thank you for flagging this, in the meantime, to ignore the "misconfiguration", you could create a `.trivyignore` file and you can pass in the following:
```
AVD-DS-0002
```
Based on the information provided in the scan:
<img width="1068" alt="Screenshot 2023-04-30 at 11 33 24" src="https://user-images.githubusercontent.com/33576047/235348401-5c1e40c4-c073-4b0c-b13b-3fd7703d33fd.png">

 if the file is in the same directory as where the trivy command is run, trivy will automatically pick it up in misconfiguration scans.

Here is the docs: https://aquasecurity.github.io/trivy/v0.41/docs/configuration/filtering/#by-finding-ids
